k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml

```diff
@@ -123,6 +123,12 @@ spec:
                     name: VH
                     recurse: true
                     title: Video Hearings Dashboard
+                - buildMonitor:
```

Comment:
looks unrelated to the change? I would separate to a separate PR normally

not a blocker

Reply:
will take on board for next PRs